runtime/experiments/payload/run.sh

```diff
@@ -140,9 +140,9 @@ process_results() {
 		awk '
 		BEGIN {
 			sum = 0
-			p50 = int('"$oks"' * 0.5)
-			p90 = int('"$oks"' * 0.9)
-			p99 = int('"$oks"' * 0.99)
+			p50 = int('"$oks"' * 0.5) + 1
```

Comment:
No clue what this is for

Reply:
I added this offset because Awk uses 1-based indices for rows, so the first row is 1, not 0. However, this ended up getting rendered moot by the checks I have for either rendering a number or `*` if there isn't enough sample data. This code will end up replaced by calls to the bash library functions in `percentiles_table.sh` when this draft PR is done.